code-challenges.js

```diff
@@ -39,13 +64,34 @@ const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+const combineArrays = (Array1,Array2) =>{
+    /* The purpose of this function is to combine two arrays and return the of COMBINED array.
+    Eg: const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+        const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+        Expected output: 9
+    */ 
+    res = Array1.concat(Array2)
+    return res.length
+}
+console.log(combineArrays(padres1984WorldSeriesRuns,padres1998WorldSeriesRuns));
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
 
 const currentCohort = "Charlie 2023"
 // Expected output: "3202 eilrahC"
 
+const revyStringy = (sentence) =>{
+    /* The purpose of this funciton is to take a string as a paramter. Then return the reverse of that string
+    Eg: const currentCohort = "Charlie 2023"
+        Expected output: "3202 eilrahC"
+    */
+    return sentence.split("").reverse().join("")
+}
+
```

Comment:
Awesome! Again, make sure your pseudo code is explaining more of the functionality rather than purpose.